testdata/golang/light/equilibriumIndices.go

```diff
@@ -0,0 +1,21 @@
+package light
+
+import "strconv"
+
+func equilibriumIndices(sequence []int) (indexList string) {
```

Comment:
this function returned a int slice before, right?

The reason the Java version returns a string is because we cannot yet properly handle ArrayLists, so I changed the implementation to return a string.

Not sure what to do here.

Reply:
Since the idea is to sync the Go and Java examples maybe we should also return a string.